servers/visual/visual_server_viewport.cpp

```diff
@@ -131,6 +131,10 @@ void VisualServerViewport::_draw_viewport(Viewport *p_viewport, ARVRInterface::E
 				if (cl->enabled && cl->texture.is_valid()) {
 					//not super efficient..
 					Size2 tsize = VSG::storage->texture_size_with_proxy(cl->texture);
+					// HACK Avoid using lights with texture of 0 size
```

Comment:
I don't think it's a hack, that's a correct way to fix it IMO.

Reply:
That's great. I updated the comment.

I thought it was kind of a hack, because it seems more like bypassing the issue (That texture with 0 size is even processed further) then actually fixing it